src/components/forms/ParentDropdown.js

```diff
@@ -0,0 +1,36 @@
+import React, { Component } from 'react'
+import "../../styles/form.css"
+
+export default class ParentDropdown extends Component {
+  onChangeHandler( event ) {
+    const { setFieldState, name, options } = this.props
+
+    setFieldState( name, event.target.value )
+    const childDropdown = options.filter( option => {
+      return option.value === event.target.value
+    })[0].field
+    setFieldState( childDropdown.name, childDropdown.defaultValue )
+  }
+
+  render() {
+    const { options, name, state } = this.props
+    let dropdownOptions = []
```

Comment:
See refactor notes above. This should be a separate function. Please use whitespace between setup and logic